lib/array.rb

```diff
@@ -0,0 +1,35 @@
+class Array
+  def faisal_inject (arg = 'empty', symbol = nil)
+
+    arithmetic_proc = symbol.to_proc if symbol.instance_of? Symbol
+    arithmetic_proc = arg.to_proc if arg.instance_of? Symbol
+
+    if arg.instance_of? Fixnum
+      return faisal_inject(arg){|k,v| arithmetic_proc.call(k,v)} if symbol.instance_of? Symbol
```

Comment:
Line is too long. [94/80]<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.